koku/masu/openapi.json

```diff
@@ -1769,6 +1769,51 @@
         ]
       }
     },
+    "/trino_ui/": {
+      "get": {
+        "summary": "Returns JSON with query output.",
+        "operationId": "TrinoUI",
+        "description": "Returns JSON with query output.",
+        "parameters": [
+          {
+            "name": "api_service",
+            "in": "query",
+            "description":"name of the trino ui api to call",
+            "required": true,
+            "schema":{
+              "type":"string",
+              "format":"string",
+              "example":"query"
```

Comment:
```suggestion
              "enum": ["query", "stats", "cluster"]
```

Reply:
👍🏿  thanks for the feedback

resolved in [this commit](https://github.com/project-koku/koku/pull/3857/commits/0b467a18e3296bd5d1350da5f06cefc77abdb35e)

is this  `/` vs `_`  for consistency with internal trino endpoints?